alerts/config.go

```diff
@@ -42,6 +46,7 @@ type Alerts struct {
 func (c Config) Validate(validator structure.Validator) {
 	validator.String("UserID", &c.UserID).Using(user.IDValidator)
 	validator.String("FollowedUserID", &c.FollowedUserID).Using(user.IDValidator)
+	validator.String("UploadID", &c.UploadID).Exists().Using(data.SetIDValidator)
```

Comment:
The first parameter should be the "JSON path" to the field so the error message has the same name with casing as the JSON field name. So:
```suggestion
	validator.String("userID", &c.UserID).Using(user.IDValidator)
	validator.String("followedUserID", &c.FollowedUserID).Using(user.IDValidator)
	validator.String("uploadID", &c.UploadID).Exists().Using(data.SetIDValidator)
```